lib/animation/animation.dart

```diff
@@ -0,0 +1,30 @@
+part of supercharged;
+
+/// Supercharged extension on [Animation] objects.
+extension Animation_ on Animation {
```

Comment:
We need generics annotation here, since `drive` function is only applikable  on `Animation<double>`